### ISSUE TITLE
Minizinha NFC Raspberry Pi

### ISSUE THREAD

**Issue by redskinsan** [pagseguromaster/plugpag#43]
Já vi um tópico falando a respeito do uso da Minizinha no Raspberry PI. Pra Minizinha NFC já funciona no Raspberry PI? Já existe uma versão q funcione no Linux? Pois no endereço https://pagseguro.uol.com.br/para-seu-negocio/solucoes-empresariais/plugpag diz q o PlugPag tem compatibilidade com a Minizinha NFC.

**Comment by backsubzero** [comment]
Espero isso a anos... acabei colocando para operar em android com tablet mesmo...

**Comment by redskinsan** [comment]
@backsubzero, é uma pena, pq RPi é uma plataforma muito usada no mundo inteiro. É estranho, pq apesar de td Android tb é Linux, se é possível implementar a criptografia em Android, pq não é implementada em Linux? Até pq um dos pontos fortes do Linux é a segurança. O @hhideki tinha comentado ser uma questão de segurança na Issue 21 em Março de 2019, será q não deu tempo de corrigir isso?

**Comment by backsubzero** [comment]
Sim, android é linux, porem tem tbm a quantidade de pessoas que usariam, somos poucos que usam RPI...
tentei rodar android na RPI e executar o plugpag, tbm não deu certo, tive outro problema, agora usam AAR no lugar de JAR, então não é mais possível executar usando JNI para interagir, resumindo eu desisti... existem outras maquinas no mercado agora e outras virão como a versão Minizinha NFC, eu já recebi uma e não consegui usar pq estou usando uma versão 3 bem velha, não consigo evoluir devido ao AAR no lugar do JAR...

**Comment by redskinsan** [comment]
Eu vou tentar implementar usando ao Moderninha Plus, o problema é q ela tem os botões de acesso ao menu e pra autoatendimento complica. Será q a PagSeguro não tem uma outra opção nesse sentido? TEF?
